### ISSUE TITLE
galaxy account name

### ISSUE THREAD

**Issue by drew-kun** [ansible/galaxy#310]
Hi everyone.

I am experiencing an issue with ansible galaxy account.

There was no option to create an account using email and password so I have created a galaxy account using my github account ('Oldname').
I have checked how it works importing 4 roles from github.
Then I have changed the name of my github account to 'new-name' and relinked it to galaxy.
But my galaxy account remains the same ('Oldname') which is not what I expected or wanted.
'My content' though now shows that I have ansible roles from the github account with new name: 'new-name/ansible-role'...
The problem is that I have some roles linked to old (e.g.: Oldname.role1) name and some roles linked to new name (e.g.: new-name.role2)

So now some of the roles are basically garbage and others (with the new account name but same old role names are ignored by galaxy and not being imported)

Questions:
1 - How can I change my galaxy account name to be the same as my new-name on github account?
2 - How can I remove all the Oldname.roles and reimport the roles with new correct name in order to keep galaxy clean and organized?

Thank you

**Comment by drew-kun** [comment]
Ok, I have re-imported the roles toggling the switch on the left of each role with new names... That's fine... But the first question is still open...

**Comment by chouseknecht** [comment]
@drew-kun 

Sorry you had to go through this in order to change your GitHub account name. I'm pretty sure we can make this easier in the future. 

Marking as a bug. We'll research, and see what we can do within Django Allauth to create some account management tools.

For now, it sounds like you solved your own problem. Is there any help you need from us right now?

Thanks!

--c

**Comment by drew-kun** [comment]
Yes. The names of roles are fixed now, but the Galaxy account name is still showing up as old, which is annoying..

**Comment by chouseknecht** [comment]
@drew-kun 

Looks like your GitHub user was originally `Drewshg312`. I updated the account record in Galaxy, so it should now appear as `drew-kun`.

**Comment by drew-kun** [comment]
Thank you very much ;)

**Comment by tnaglo** [comment]
@chouseknecht 
Sorry to bother you, but I have done the same (bad) thing-> changed my username:
from Naglik to tnaglo

I updated and imported my roles from the new github account (name), galaxy still shows the old one. can you please fix that?

**Comment by drew1kun** [comment]
Hi @chouseknecht! Thank you so much for your help. I am currently did the same thing again - changed my name from drew-kun to drew1kun. It is the last time, I promise! Could you please help me to change my owner name to `drew1kun` on Galaxy and remove the old empty `drew-kun` namespace as it seems impossible to delete it as it is somehow tied to a Galaxy account name...
